docs/source/installation/translations.rst

```diff
@@ -0,0 +1,82 @@
+.. _install-dev:
+
+Translations
+************
+
+Indico comes with a number of languages by default. In release 2.3, those are: 
+English (default), French, Portuguese, Spanish and Chinese (in the order of integration).
+Additional languages are being prepared on the Transifex platform.
+
+In order to use (partially) existing translations from Transifex or to contribute 
+translations, you need to register with the
+`Indico project on the Transifex platform <https://www.transifex.com/indico/>`_.
+
+Additional Translations
+=======================
+
+This is a guide 
+(courtesy `pedrovsky <https://talk.getindico.io/t/including-a-new-language-in-indico/1864/5>`_) 
+to set up an Indico instance with a new language. 
+It is useful for translators when applying their revisions to verify how the translation looks in production
+or for administrators who just want to lurk at the incubated translation embryos.
+
+Create your own Indico instance
+-------------------------------
+
+This could be in our own computer, in a shared machine or even your organisation, if the latter gives you permission.
+
+For creating your own Indico instance, we provide two different guides:
+The first one is for a `production system <../installation/production>`_, 
+it will prepare Indico to be served to users and used in all the different purposes you may have besides translations. 
+The second is `development <../installation/development>`_ a light-weight, 
+easier to set up, version oriented to testing purposes, that should not be exposed to the public.
+
+However, for the purpose of translation **development** or **testing** which are the purpose of this guide, 
+we recommend and only provide the steps when using the development version.
+
+Installing the `transifex client <https://docs.transifex.com/client/installing-the-client>`_
+--------------------------------------------------------------------------------------------
+
+Get an API token
+----------------
+
+Go `here <https://www.transifex.com/user/settings/api/>`_ and generate your API token. 
+Afterwards, you should run the command ``tx init --skipsetup``. 
+It will request the token you just copied from the previous settings and save it locally so you can start using transifex locally.
+If you do not know how to run this command, 
+please refer to the guide `here <https://docs.transifex.com/client/init>`_.
+
+Install the translations
+------------------------
+
+Navigate to the ``src`` folder inside the Indico path.
```

Comment:
Maybe better to explicitly mention `~/dev/indico/src/` - someone who's familiar with their local dev setup won't need it but for someone who isn't it's probably better to be clear on where to go.